app/cdap/components/PipelineDiff/DiffInfo/DiffInfoTable.tsx

```diff
@@ -79,14 +81,20 @@ export const DiffInfoTable = ({ diffIndicator, pluginProperties }: IDiffInfoTabl
       <Table stickyHeader size="small">
         <TableHead>
           <TableRow>
-            <TableCell>Name</TableCell>
+            <TableCell>{T.translate(`${I18N_PREFIX}.table.name`).toString()}</TableCell>
```

Comment:
I don't think `toString()` is necessary

Reply:
Since the child type can be ReactNode, you don't need to have toString, check other places as well.

Reply:
done